Mathport/Syntax/Translate.lean

```diff
@@ -57,16 +57,14 @@ def AST3toData4 (path : Path) : AST3 → M Data4
     let fmt ← liftCoreM $ PrettyPrinter.format Parser.Module.header.formatter $
       mkNode ``Parser.Module.header #[mkOptionalNode prel, mkNullNode imp]
     let commitInfo := (← read).config.commitInfo
-    let msg : String :=
-      "! This file was ported from Lean 3 source module " ++ path.mod3.toString ++ "\n" ++
-      (if let some ci := commitInfo
-        then
-        "! " ++ ci.repo ++ " commit " ++ ci.fileRevs.findD (path.mod3.toFilePath.toString ++ ".lean") ci.commit ++ "\n" ++
-        "! Please do not edit these lines, except to modify the commit id\n" ++
-        "! if you have ported upstream changes.\n"
-        else "")
-    printFirstLineComments (some msg)
+    printFirstLineComments
     printOutput fmt
+    -- todo: use the pretty-printer?
+    if let some ci := commitInfo then
+      let commit := ci.fileRevs.findD (path.mod3.toFilePath.toString ++ ".lean") ci.commit
+      printOutput f!"#align_import {path.mod3} from {repr ci.repo}@{repr commit}"
+    else
+      printOutput f!"#align_import {path.mod3}"
```

Comment:
I think this needs some newlines before and after. What's the intended style?

Reply:
I guess blank line before and after. Does `printOutput` print a trailing newline?

Reply:
Added: indeed, the current output is:
```
#align_import oxford_invariants.«2021summer».week3_p1 from "leanprover-community/mathlib"@"08b081ea92d80e3a41f899eea36ef6d56e0f1db0"/-!
# module doc
```
when it shoudl be
```
#align_import oxford_invariants.«2021summer».week3_p1 from "leanprover-community/mathlib"@"08b081ea92d80e3a41f899eea36ef6d56e0f1db0"

/-!
# module doc
```

Reply:
does it have a blank line before any imports?

Reply:
Yes, sorry; edited above.